stats/config/utils/find_free_timeslot.py

```diff
@@ -0,0 +1,262 @@
+import tkinter as tk
```

Comment:
nice, like it! Could you please add description of that tool anywhere, like 
* what is it
* how to use it (install deps and run)

Reply:
makes sense
btw it's fully written by claude 🤩

Reply:
done

Reply:
the interface is fully visual btw, so it's very trivial